src/components/graph/menus/modification-list-item.js

```diff
@@ -34,7 +35,18 @@ export const ModificationListItem = ({ modification, ...props }) => {
     return (
         <>
             <ListItem {...props}>
-                <OverflowableText text={getLabel()} />
+                <OverflowableText style={{ flexGrow: '1' }} text={getLabel()} />
+                <IconButton
+                    onClick={() => onDelete(modification.uuid)}
+                    size={'small'}
+                    style={{ alignSelf: 'right' }}
```

Comment:
You will certainly need padding or margin later then maybe use a style class for the button and for the `OverflowableText` like in other components like that

Reply:
done